### PR TITLE
GAWB-1818: Add call caching blog post url

### PR DESCRIPTION
**Pull request by ansingh7115**
screen shot 2017-04-05 at 12 16 43 pm

Hovering over the icon gives you an explanation. Clicking on it takes you [here](http://gatkforums.broadinstitute.org/firecloud/discussion/9313/call-caching).

This is a consistent with the Firecloud User Guide link that's always on the top right of every page on the site.

Config change added to release ticket

- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * Add notes on what you've tested
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [x] Tell the tech lead (TL) that the PR exists if they wants to look at it
- [x] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [x] **TL** sign off
- [x] **LR** sign off
- [x] **Product Owner** sign off
- [ ] **Assign to submitter** to finalize
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: src/cljs/main/broadfcui/page/workspace/method_configs/launch_analysis.cljs [hunk 1]
```diff
@@ -82,7 +82,7 @@
    [:div {:style {:marginTop "1em"}}
     [comps/Checkbox
     {:ref "callCache-check"
-     :label "Use Call Caching"
+     :label [:span {:style {:marginBottom "0.8em"}} "Use Call Caching. " [:a {:href (config/call-caching-guide-url) :target "_blank"} "Learn More"]]
```
Comment (helgridly) [inline src/cljs/main/broadfcui/page/workspace/method_configs/launch_analysis.cljs]: check with @vdauwera for the wording here?

Comment (MattPutnam) [inline src/cljs/main/broadfcui/page/workspace/method_configs/launch_analysis.cljs]: I would suggest using the info box, see `common/render-info-box`.

Comment (vdauwera) [reply src/cljs/main/broadfcui/page/workspace/method_configs/launch_analysis.cljs]: Are there any established conventions for how to display such help elements? And would it be possible to include a screenshot in the PR? Without that it's hard to visualize the result.

**Comment by ansingh7115** [comment]
@vdauwera  added a screenshot in the PR description above.
Ready for re-review

**Review by helgridly — APPROVED** [review]

**Comment by MattPutnam** [comment]
Since this shares almost everything with the existing one, let's make it a reusable component.  It's stateless, so it doesn't have to be a React component.

**Comment by MattPutnam** [comment]
👍

**Pull request merged**